src/memote/config.jl

```diff
@@ -340,7 +341,7 @@ end
 network_config = NetworkConfig(
     condition_number = 1e9,
     cycle_tol = 1e-3,
-    blocked_tol = 1e-3,
+    blocked_tol = 1e-8,
```

Comment:
this is a bit too strict no? (with float32 solvers it's just going to fail everytime)

Reply:
It misses fluxes in iML1515 if it is that low... need to be strict :/

Reply:
:sob: